client/app/ui-components/inline-color-alert/inline-color-alert.component.js

```diff
@@ -0,0 +1,26 @@
+(() => {
+    "use strict";
+
+    class OuiInlineColorAlertController {
+
+        getClass () {
+            return `oui-inline-color-alert__${this.type}`;
+        }
+    }
+
+    angular.module("managerApp")
+        .component("ouiInlineColorAlert", {
```

Comment:
I like that!
I know CX is working on guidelines for _statuses_
Depending on what they decide, we'll move this component to oui-angular (in a few weeks?)

Reply:
ok... we are using this component in 2 pages as of now